### PR TITLE
Tournament player titles

### LINKED ISSUE #1814: Fide title display next to username in tournaments
I think that it will be great in mobile app if in tournaments i can see the fide title of a user next to the username.

### PR DESCRIPTION
**Pull request by jas14**
Closes #1814. Compare screenshot to the one in the issue.

### CODE REVIEW

**Comment by Baikalcadillac** [comment]
Small detail I’ve noticed - website version have FIDE titles below winner’s cup as well. 

4491E91E-258A-4398-8A65-68FE1B7FE9F7

**Comment by jas14** [comment]
Thanks for the suggestion @Baikalcadillac , updated!

File: src/ui/tournament/detail/tournamentView.tsx [hunk 1]
```diff
@@ -274,23 +274,34 @@ function renderNavButton(icon: string, isEnabled: boolean, action: () => void) {
...
+          {`${player.name} (${player.rating})`}
+        </span>
+        <span className={`playerTeam ttc-${ttc}`}> {teamName ?? ''} </span>
       </div>
       <div className={'tournamentPoints ' + (player.sheet.fire ? 'on-fire' : 'off-fire')} data-icon="Q">
         {player.score}
       </div>
     </li>
-  )
+  ) as Mithril.Child
```
Comment (veloce) [inline src/ui/tournament/detail/tournamentView.tsx:304]: Why casting here? It should be not necessary. It's preferable to put the function return type explicitly then.

Comment (jas14) [reply src/ui/tournament/detail/tournamentView.tsx:304]: Typescript doesn't seems to recognize JSX as Mithril.Vnode / Mithril.Child. The cast was to remove the "`any` return type" lint warning. A better long-term solution will be to tell Typescript about JSX or convert it all to `h` calls but that's a different PR :)

Comment (veloce) [reply src/ui/tournament/detail/tournamentView.tsx:304]: ah ok thanks. I didn't notice this warning actually

**Pull request merged**